README.md

```diff
@@ -833,32 +833,56 @@ SubmissionField is an interface that represents a mapping between your applicati
 values. SubmissionField's are used by the FFB library during PDF generation to map your application'
 s
 input values to the correct PDF Fields. There are 3 types of SubmissionFields:
-
-| SubmissionField Implementation | Description                                                                                                                                                                                                             | Constructor                                                                                           | Examples |
-|--------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------|----------|
-| SingleField                    | For single value input fields. Represents a one to one mapping between an input in your application such as a text, radio or drop down field and it's value. Can include an iteration if the input is within a subflow. | Params: `String` input name, `String` input value, `Int` iteration number (for subflows, can be null) | **       
-
-Outside of a
-subflow**: <br> new SingleField("exampleInputName", "exampleInputValue", null) <br><br> **Inside of
-a
-subflow**: <br> new SingleField("exampleInputName", "exampleInputValue", 1)
-|
-| CheckboxField | For checkbox input fields. Represents a one to many mapping between a checkbox
-input and an array of it's many potential values. Can iclude an iteration if the input is within a
-subflow. | Params: `String` input name, `ArrayList<String>` input value, `Int` iteration
-number `Int` iteration number (for subflows, can be null) | **
-Outside of a
-subflow**: <br> new CheckboxField("exampleInputName", List.of("exampleValueOne", "exampleValueTwo"),
-null) <br><br> **
-Inside of a
-subflow**: <br> new CheckboxField("exampleInputName", List.of("exampleValueOne", "exampleValueTwo"),
-
-1) |
-                                                                                                | DatabaseField | For fields from database columns. Represents a mapping between database fields
-   such as `submittedAt`, `submissionId`, etc and their values. Does not include an iteration. |
-   Params: `String` database column name, `String` database field value | new DatabaseField("
-   submittedAt", "exampleSubmittedAtValue")
-   |
+<table>
```

Comment:
This renders really nicely.  Does it allow for you to indent the HTML to make it more readable?  Not sure if that's possible or not.